src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -1374,6 +1374,11 @@ braid_structured(index_t npts_x,
                                  npts_z,
                                  res["coordsets/coords"]);
 
+    if (npts_y < 1 && npts_z < 1)
+    {
+        res["coordsets/coords/type"] = "rectilinear";
+    }
+
```

Comment:
Not sure I understand this change.

braid_structured aims to create implicit logical topology with on an explicit coordset.

Reply:
woops.  I mixed up rectilinear and structured.  Just trying to keep the amount of allocation to a minimum.